fms_core/models/sample.py

```diff
@@ -255,25 +255,17 @@ def add_error(field: str, error: str):
 
         self.normalize()
 
-        biospecimen_type_choices = (Sample.BIOSPECIMEN_TYPE_NA_CHOICES if self.extracted_from
```

Comment:
This validation disappeared. The foreign key prevent use of goofy sample kind but extracted sample must still be of a subset of sample kind ... Maybe in a future pass we can see if we cannot define sample kind more robustly to do these validations without relying on Constant subsets of sample kinds.